eth/contracts/AbstractMost.sol

```diff
@@ -40,6 +41,8 @@ abstract contract AbstractMost is
 
     address public wrappedAzeroAddress;
 
+    ITransferLimit public transferLimit;
```

Comment:
Doesnt this makes l2 and l1 most unupgradable?

Reply:
Didnt we want limits only for l1 most since we have fees in both direction on the l2?

Reply:
Agree with the second comment. About the first one, I've just tried upgrading to L1, and it worked. Not sure if this line breaks L2 upgrade, have not checked.

Reply:
You are right l1 is still fine but l2 should break since it have fields in the mostl2.sol

Reply:
Let's then remove most2.sol entry and we should be good

Reply:
we are a littlebit to late with https://eips.ethereum.org/EIPS/eip-7201 since l1 most was already deployed

Reply:
> Let's then remove most2.sol entry and we should be good 
We would need to remove also new field from abstractmost and move it to l1most

Reply:
Done - @Marcin-Radecki want to try the upgrade again?

Reply:
Yes, it works.